folium/__init__.py

```diff
@@ -67,7 +67,7 @@
     
     - Install an older version of folium:
     
-     $ pip install 'folium<9.0'
+     $ pip install 'folium<0.9.0'
```

Comment:
We actually should be at version `9.0` by now :smile: 

Maybe we can make the Python 3 only our 1.0!

Reply:
I was thinking about what we’d need for a 1.0 version the other day, let’s talk about that some time.